src/rpdk/core/contract/resource_client.py

```diff
@@ -458,8 +457,7 @@ def compare_model(self, inputs, outputs, path=()):
             "All properties specified in the request MUST "
             "be present in the model returned, and they MUST"
             " match exactly, with the exception of properties"
-            " defined as writeOnlyProperties in the resource schema \n Request Model : %s \n Returned Model : %s \n"
```

Comment:
formatting and functional changes should probably be separate, but assuming this was done automatically so should be good

Reply:
Yea some of this was the result of passing pylint updates.  I was debating separating it.